ci-operator/step-registry/ibmcloud/provision/vpc/endpoints-gateway/ibmcloud-provision-vpc-endpoints-gateway-commands.sh

```diff
@@ -128,7 +207,8 @@ for srv in "${srv_array[@]}"; do
         fi
         target=$(findTarget "${allTargetsFile}" "${real_srv_endpoint_url}")
         if [[ -z "${target}" ]]; then
-            echo "ERROR: Did not find out endpoint gateway target"
+            echo "ERROR: Did not find out endpoint gateway target of $srv"
+            cat ${allTargetsFile} > "${ARTIFACT_DIR}/ep_targets.json"
```

Comment:
If you like to keep `allTargetsFile`, you can directly change the line 108 to `ARTIFACT_DIR`

Reply:
To save the space in ${ARTIFACT_DIR}, just save it to ${ARTIFACT_DIR} when fail to find the target.

Reply:
You already have the line to dump out them into `ARTIFACT_DIR` when failed. It make no much difference that always dump out them.